openmrs/apps/clinical/formConditions.js

```diff
@@ -382,5 +382,1566 @@ Bahmni.ConceptSet.FormConditions.rules = {
 			conditions.hide.push("Lymphadenopathy");
 		}
 		return conditions;
+	},
+	'Vaginal Cervical smear Clue cells': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};
+		var value = formFieldValues['Vaginal Cervical smear Clue cells'];
+		if (value && value !== "") {
+			conditions.show.push("Vaginal Cervical smear Clue cells Test Date");
+		} else {
+			conditions.hide.push("Vaginal Cervical smear Clue cells Test Date");
+		}
+		return conditions;
+	},
+	'Vaginal Cervical smear Yeast cells per field': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};
+		var value = formFieldValues['Vaginal Cervical smear Yeast cells per field'];
+		if (value && value !== "") {	
+			conditions.show.push("Vaginal Cervical smear Yeast cells per field Test Date");
+		} else {	
+			conditions.hide.push("Vaginal Cervical smear Yeast cells per field Test Date");
+		}	
+		return conditions;	
+	},
+	'Vaginal Cervical smear Epithelial cells per field': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};		
+		var value = formFieldValues['Vaginal Cervical smear Leucocytes per field'];
```

Comment:
@lauravignoli Did you mean `Vaginal Cervical smear Epithelial cells per field` here?

Reply:
Fixed